.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -9,12 +9,27 @@ body:
         Thank you for contributing to iTwinUI by creating an issue!
         Make sure you first do a search for existing issues so that this is is not a duplicate.
         Also visit our [FAQ](https://github.com/iTwin/iTwinUI/wiki/FAQ).
+  - type: dropdown
+    attributes:
+      label: Are you using the latest version of iTwinUI?
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true
+    id: version
+  - type: textarea
+    attributes:
+      label: Version Details
+      description: If you are not using the latest version, please specify the version you are using.
+    validations:
+      required: false
```

Comment:
I think we could instead show a checkbox at the end?

[ ] I am using the latest version of iTwinUI.

Reply:
Does this change okay? [commit](https://github.com/iTwin/iTwinUI/pull/2232/commits/4b0bc8baf4d81c60312a3079c921cdb0b083c023). Then users can specify their current version, if not the latest, in the last "Anything else?" text box.